### PR TITLE
No need for BucketInfo to be virtual just to make it printable.

### PR DESCRIPTION
**Pull request by baldersheim**
@vekterli PR

### CODE REVIEW

File: storageapi/src/vespa/storageapi/message/bucket.cpp [hunk 1]
```diff
@@ -552,7 +548,7 @@ NotifyBucketChangeCommand::print(std::ostream& out, bool verbose,
                                  const std::string& indent) const
 {
     out << "NotifyBucketChangeCommand(" << getBucketId() << ", ";
-    _info.print(out, verbose, indent);
+    out << _info.toString();
```
Comment (vekterli) [inline storageapi/src/vespa/storageapi/message/bucket.cpp:551]: Could be simplified down to just `out << _info;` (unless we want stringification to be explicit)

**Review by vekterli — APPROVED** [review]
:+1: the number of pointless vtables that have historically been scattered around due to `Printable` and its friends makes me think that maybe we should remove the thing entirely.

File: storageapi/src/vespa/storageapi/message/bucket.cpp (hunk 1 above)
Comment (baldersheim) [reply storageapi/src/vespa/storageapi/message/bucket.cpp:551]: Correct, will do

Comment (baldersheim) [reply storageapi/src/vespa/storageapi/message/bucket.cpp]: Fixed

**Pull request merged**